Chutzpah/Models/ChutzpahTestSettingsFile.cs

```diff
@@ -47,7 +47,8 @@ public enum Engine
     {
         Phantom,
         Chrome,
-        JsDom
+        JsDom,
+        HeadlessChromium

```

Comment:
@zhangyiatmicrosoft This feels confusing since Chrome setting listed here is also headless. I think in short you just want a way to force server not to run. I have server run since in webtests for chrome and jsom it really is the right choice. But in this case since you really want to force it off, maybe adding a different special setting that is like: **ForceServerModeOff**? Something most people won't use but is there in these special situations.

Reply:
yes, that is right, need a chrome+ForceServerOff. Let me get back with you on that.